sql/sys_vars.cc

```diff
@@ -1513,7 +1513,9 @@ static Sys_var_bit Sys_log_slow_admin_statements(
        "to the slow log if it is open.  Resets or sets the option 'admin' in "
        "log_slow_disabled_statements",
```

Comment:
I think it should mention that it is deprecated in the text here. We seem to do that with most other options.

Reply:
Done. Also there is no 'admin' in 'log_slow_disabled_statements'. Corrected that as well.